lib/tentacat/repositories/deploy_keys.ex

```diff
@@ -3,7 +3,8 @@ defmodule Tentacat.Repositories.DeployKeys do
   alias Tentacat.Client
 
   @moduledoc """
-  The Repository deploy keys API allows repository admins to manage the deploy keys for a repository.
+  The Repository deploy keys API allows repository admins to manage the deploy
+  keys for a repository.
```

Comment:
why the new line 🤔

Reply:
Line wrapping follows the existing file line length.